src/utilities/theme/types.ts

```diff
@@ -11,25 +9,31 @@ export type ThemeLogo = {
   accessibilityLabel?: string;
   /** Number of pixels wide the logo image is */
   width?: number;
-} | null;
-
-export interface ThemeColor {
-  [key: string]: string;
-}
+};
 
-export interface TopBar extends ThemeColor {
-  background: string;
+// The value that is passed into the ThemeProvider
+export interface ThemeConfig {
+  /** Sets the logo for the top bar and contextual save bar components*/
+  logo?: ThemeLogo;
+  colors?: {
+    /** Sets the background color of the top bar component. Complimentary and typography colors are determined programmatically */
+    topBar?: Record<string, string>;
```

Comment:
It'd be great to if we could identify the valid keys within topBar and identify them here

Reply:
Going to leave this one as-is.